adagrams/game.py

```diff
@@ -1,11 +1,114 @@
+import random
+
 def draw_letters():
-    pass
+
+    letter_probability = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    
+    # Create pool of letters with frequency as outlined in letter_probability dict
+    pool_of_letters = []
+    for letter in letter_probability:
+        pool_of_letters.extend([letter] * letter_probability[letter])
+
+    # Choose a hand of 10 letters
+    hand_of_letters = []
+
+    while len(hand_of_letters) < 10:
```

Comment:
The body of the while loop definitely works and doesn't mutate any mutable objects, which I do like! In this case, though, mutation might actually simplify things. Since you have a list of all the possible letters, you can use `random.randint` to select a random index based on `pool_of_letters` list size (like you already do), get the letter from `pool_of_letters` at that random index, add it the `hand_of_letters`, then remove it from the `pool_of_letters`, e.g.
```python
    random_index = random.randint(0, len(pool_of_letters) - 1)
    hand_of_letters.append(pool_of_letters[random_index])
    pool_of_letters.pop(random_index)
```
It's a little simpler and has less conditional logic this way.